docs/rules/no-extraneous-dependencies.md

```diff
@@ -31,7 +37,13 @@ If you are using JavaScript configuration (e.g., `.eslintrc.js`), then you can u
 "import/no-extraneous-dependencies": ["error", {"devDependencies": /test|spec/}]
 ```
 
-When using a regular expression, the setting will be activated if the name of the file being linted matches the given regular expression. For example, the above configurations will allow the import of `devDependencies` in files whose names include `test` or `spec`.
+When using a glob or regular expression, the setting will be activated if the name of the file being linted matches the given glob or regular expression. In addition, you can use an array to specify multiple globs or regular expressions:
+
+```js
+"import/no-extraneous-dependencies": ["error", {"devDependencies": [/test/, '/spec/', '*.test.js', '*.spec.js']}]
+```
+
+When using an array of globs or regular expressions, the setting will be activated if the name of the file being linted matches a single glob or regular expression in the array.
```

Comment:
> matches a single glob or regular expression in the array

Hmm... Not sure this part works. Globs as an array allows you to specify multiple things `['test/**/*.js', 'src/**/*.js']`, but also to forbid elements allowed by previous "expressions": `['test/**/*.js', 'src/**/*.js', '!fixtures']`. Here `<root>/test/fixtures/a.js` should not work, whereas from your docs (and guessing implementation), it would.

(This is a glob right? Or am I confusing terms?)